src/managers/GitIgnoreManager.js

```diff
@@ -6,15 +6,14 @@ import { join } from '../utils/join.js'
 
 // I'm putting this in a Manager because I reckon it could benefit
 // from a LOT of cacheing.
-
-// TODO: Implement .git/info/exclude
-
 export class GitIgnoreManager {
   static async isIgnored({ fs, dir, gitdir = join(dir, '.git'), filepath }) {
     // ALWAYS ignore ".git" folders.
     if (basename(filepath) === '.git') return true
     // '.' is not a valid gitignore entry, so '.' is never ignored
     if (filepath === '.') return false
+    // Load exclusion rules from the project exclude file (.git/info/exclude)
+    const excludes = await fs.read(join(gitdir, 'info', 'exclude'), 'utf8')
```

Comment:
I would add check if the file exists first.

Reply:
Fixed in dcad82d.